substra/sdk/client.py

```diff
@@ -461,9 +467,7 @@ def add_composite_traintuple(self, data, timeout=False, exist_ok=False):
         If `exist_ok` is true, `AlreadyExists` exceptions will be ignored and the
         existing asset will be returned.
         """
-        res = self._add(assets.COMPOSITE_TRAINTUPLE, data, timeout=timeout,
-                        exist_ok=exist_ok,
-                        permissions_field='out_model_trunk_permissions')
+        res = self._add(assets.COMPOSITE_TRAINTUPLE, data, timeout=timeout, exist_ok=exist_ok)
```

Comment:
There is a permission field in this case: `out_model_trunk_permissions`. It should also be populated with a default value if not present.